fastembed/embedding.py

```diff
@@ -647,6 +683,19 @@ def embed(
                 embeddings, attn_mask = batch
                 yield from normalize(self.mean_pooling(embeddings, attn_mask)).astype(np.float32)
 
+    def split_text(self, text: str, chunk_size: Optional[int] = None, chunk_overlap: Optional[int] = None) -> List[str]:
+        """Splits text into chunks based on the tokenizer encoding size.
+
+        Args:
+            text (str): The text to split.
+            chunk_size (Optional[int], optional): Maximum size of chunks based on the tokenizer encoding.
+            chunk_overlap (Optional[int], optional): Allowed overlap in characters between chunks.
+
+        Returns:
+            List[str]: The list of strings.
+        """
+        return self.model.split_text(text, chunk_size=chunk_size, chunk_overlap=chunk_overlap)
```

Comment:
Looks like we're repeating this wrapper function across classes. Would you want to add this to the abstract class upstream from which other classes can inherit and get access to this?

Reply:
https://github.com/qdrant/fastembed/pull/80/commits/0641f6fee0348f2e1dfb6b2d108e74795a582c5d